presto-docs/src/main/sphinx/functions/comparison.rst

```diff
@@ -198,3 +198,63 @@ Examples::
     SELECT * FROM (VALUES ('a%c'), ('%cd'), ('cde')) AS t (name)
     WHERE name LIKE '%#%%' ESCAPE '#'
     --returns 'a%c' and  '%cd'
+
+
+
+## Row comparison: IN
+
+The `IN` comparison operator in SQL is used to compare a value with a list of literal values that have been specified.
+The `IN` operator returns true if the value matches any of the literal values in the list.The `IN` operator can be
+used to compare values with the following patterns.It can fetch records according to multiple values specified in
```

Comment:
```suggestion
The `IN` operator returns true if the value matches any of the literal values in the list. The `IN` operator can be
used to compare values with the following patterns. It can fetch records according to multiple values specified in
```